node-setup/080_register_stakepool.md

```diff
@@ -172,4 +214,9 @@ will output your poolID. You can then check for the presence of your poolID in t
 
     cardano-cli shelley query ledger-state --testnet-magic 42 | grep poolPubKey | grep <poolId>
 
+or 
+
+    cardano-cli shelley query ledger-state --testnet-magic 42 \
+    | jq '._delegationState._pstate._pParams.<poolid>' 
```

Comment:
This can only be a temporary hack. Dumping the entire ledger state is for debugging only.